elements/urth-viz-pie/urth-viz-pie.html

```diff
@@ -31,6 +31,17 @@
     Polymer({
         is: 'urth-viz-pie',
 
+        properties: {
+            /**
+             * Allows user to select multiple slices.
+             */
+            multiSelect: {
```

Comment:
No need for property here once it is moved to `elements/urth-viz-chart/urth-viz-chart-common-behavior.html`.